src/ui/public/doc_table/components/table_row/cell.html

```diff
@@ -10,4 +10,11 @@
 %>
 <td <%= attributes %>>
   <%= formatted %>
+  <span class="table-cell-filter">
+    <% if (filterable) { %>
+      <i>&nbsp;</i>
```

Comment:
What does this element do? I tried to removing it and it didn't seem to have an effect.